docs/protocol/core/v3.0.rst

```diff
@@ -1422,7 +1422,7 @@ Let "+" be the string concatenation operator.
 **Discover all nodes in a hierarchy**
 
     To discover all nodes in a hierarchy, one can call
-    ``list("meta/")``. All keys represent either explicit group or
+    ``list_prefix("meta/root/")``. All keys represent either explicit group or
```

Comment:
@alimanfoo `list` doesn't accept any parameter, so I guess we should use `list_prefix`, and I guess that `/root` was missing?

Reply:
I think maybe just `list()` ? 

```
``list`` - Retrieve all `keys` in the store.

    | Parameters: none
    | Output: set of `keys`
```

Or do we get rid of list and say that it should be constructed from list_prefix ?

Reply:
I actually implemented `list()` as `list_prefix("")`, but it's also good to keep `list()` as a shortcut.
But we can't use `list()` to discover all the nodes in a hierarchy, since we need to prefix with `"meta/root/"`.